### DISCUSSION TITLE
How to get the latitude and longitude .

### DISCUSSION THREAD

**Discussion by aungnyithit-21** [inocan-group/vue3-google-map#239] [Q&A] [answered]
Hello Sir , 
I want to get the latitude and longitude  when I click on the map in state of vue . How can  I do that?

**Comment by HusamElbashir** [comment, marked as answer]
Hi @aungmyaethit

You can listen to a `click` event on the map. Example:
https://stackblitz.com/edit/vitejs-vite-4tdpur?file=src%2Fcomponents%2FHelloWorld.vue&terminal=dev

You get a `MapMouseEvent` in your event handler that contains `latLng` which has `lat` and `lng` getter functions:
https://developers.google.com/maps/documentation/javascript/reference/map#MapMouseEvent

**Comment by aungnyithit-21** [comment]
Hi @HusamIbrahim 

Thank you so much for  answer, It really support me.
